insights/core/filters.py

```diff
@@ -46,37 +51,49 @@
 ENABLED = parse_bool(os.environ.get("INSIGHTS_FILTERS_ENABLED"), default=True)
 
 
-def add_filter(ds, patterns):
+def add_filter(component, patterns):
```

Comment:
I've changed `ds` to `component` here, if you think that might be an issue I can change that? But after looking around some, I didn't see any explicit references to `ds`.